app/src/main/res/layout/search_widget_large.xml

```diff
@@ -7,10 +7,12 @@
               android:layout_width="match_parent"
               android:layout_height="match_parent">
 
-    <RelativeLayout android:layout_width="256dp"
+    <RelativeLayout android:layout_width="match_parent"
                     android:layout_height="50dp"
                     android:background="@drawable/rounded_white_corners"
-                    android:layout_gravity="center">
+                    android:layout_gravity="center"
```

Comment:
I believe this line has no effect inside a RelativeLayout.

Reply:
Also, marginLeft and marginRight should be replaced by marginStart and marginEnd across the board to respect bidirectional language layouts. Likewise, alignParentRight should be alignParentEnd, for example.

Reply:
I just tried with and without layout_gravity="center", and this line seems to be needed to center the actual widget relative to its cells occupied.